lib/absinthe/relay/node/parse_ids.ex

```diff
@@ -229,7 +229,17 @@ defmodule Absinthe.Relay.Node.ParseIDs do
   @spec find_schema_root!(Absinthe.Type.Field.t(), Absinthe.Resolution.t()) ::
           {{Absinthe.Type.Field.t() | Absinthe.Type.Argument.t(), String.t()},
            (String.t() -> String.t())}
-  defp find_schema_root!(field, resolution) do
+  defp find_schema_root!(
```

Comment:
pattern matches on `find_schema_root!` so that it will identify if it is the parent mutation by matching on the 
```
%{
           __private__: [
             absinthe_relay: [
               payload: {:fill, _},
               input: {:fill, _}
             ]
           ]
         }
```
structure.

The other pattern-match function is for the child fields.